components/SwitchLang.tsx

```diff
@@ -48,18 +55,40 @@ const SwitchLang = () => {
           <Menu.Items className="absolute top-0 right-0 z-20 mt-8 w-28 divide-y divide-gray-900 overflow-auto rounded border border-gray-900/10 bg-white py-1 shadow-lg focus:outline-none dark:border-gray-500/30 dark:bg-gray-900 dark:text-white">
             {locales!.map(locale => (
               <Menu.Item key={locale}>
-                <CustomLink key={locale} href={{ pathname, query }} as={asPath} locale={locale}>
+                <CustomLink
+                  key={locale}
+                  href={{ pathname, query }}
+                  as={asPath}
+                  locale={locale}
+                  onClick={() => setCookie('NEXT_LOCALE', locale, { path: '/' })}
+                >
                   <div className="m-1 cursor-pointer rounded px-2 py-1 text-left text-sm font-medium hover:bg-blue-50 hover:text-blue-700 dark:hover:bg-blue-600/10 dark:hover:text-blue-400">
                     {localeText(locale)}
                   </div>
                 </CustomLink>
               </Menu.Item>
             ))}
+            <Menu.Item key="auto">
+              <CustomLink
+                key="auto"
+                href={{ pathname, query }}
+                as={asPath}
+                locale={false}
+                onClick={() => removeCookie('NEXT_LOCALE', { path: '/' })}
+              >
+                <div className="m-1 cursor-pointer rounded px-2 py-1 text-left text-sm font-medium hover:bg-blue-50 hover:text-blue-700 dark:hover:bg-blue-600/10 dark:hover:text-blue-400">
+                  {
+                    // t('Automatic')
+                    t(localeText('auto'))
```

Comment:
Shouldn't have this option here, I don't see any other website with an 'Auto' language option. I think we can keep this in the dark and not have a selectable option here.

Reply:
Indeed. I will remove it after I get home tonignt.